storage/Cargo.toml

```diff
@@ -23,7 +23,7 @@ leaky-bucket = "0.12.1"
 libc = "0.2"
 log = "0.4.8"
 nix = "0.24"
-reqwest = { version = "0.11.11", features = ["blocking", "json"], optional = true }
+reqwest = { version = "0.11.14", features = ["blocking", "json"], optional = true }
```

Comment:
@ccx1024cc I submitted a PR earlier to fix RUSTSEC-2023-0018, but unfortunately it was not merged before v2.2.0, you could cherry-pick this commit #1123 instead.

Reply:
> 

Thanks~~~~~Missed